geode-docs/tools_modules/geode_api_compatible-with-redis.html.md.erb

```diff
@@ -91,8 +90,8 @@ exactly as expected.
 -   **Strings**: BITCOUNT, BITOP, BITPOS, DECRBY, GETBIT, GETSET, MSET, MSETNX, PSETEX, SETBIT, SETEX, SETRANGE
 
 
-If you already have some Geode servers running with Redis enabled, you can execute the following
-command with gfsh to enable unsupported commands:
+If you already have some Geode servers running with Redis compatibility enabled, you can execute the
```

Comment:
If you already have some Geode servers running that are compatible with Redis,  you can execute the...